DataCollection/src/Domain/Interfaces/Persistence/ICrudSet.cs

```diff
@@ -0,0 +1,11 @@
+﻿using FlightNode.Common.BaseClasses;
+using System.Data.Entity;
+
+namespace FlightNode.DataCollection.Domain.Interfaces.Persistence
```

Comment:
It was embedded in some other file previously, instead of being in its own file.